src/compile/common.ts

```diff
@@ -56,8 +56,8 @@ export function applyColorAndOpacity(p, model: UnitModel) {
     applyMarkConfig(p, model, STROKE_CONFIG);
   }
 
-  let colorValue;
-  let opacityValue;
+  let colorValue: {scale: string, field: {prefix?: 'rank'}} | {value: string | number | true};
+  let opacityValue: {scale: string, field: {prefix?: 'rank'}} | {value: string | number | true};;
```

Comment:
Thanks! I wasn't sure what type these were supposed to be (they were being assigned either object literals or things typed 'any', so it wasn't clear (and I just made a type of the object literals).